### PR TITLE
San Mateo High First Meet / Club Fair Documentation

### PR DESCRIPTION
**Pull request by saescapa**

### CODE REVIEW

**Comment by MaxWofford** [comment]
@saescapa It looks like one of the tests have failed: https://circleci.com/gh/hackedu/hackedu/270

**Comment by MaxWofford** [comment]
Cherry picked onto master at https://github.com/hackedu/hackedu/commit/80e8de4e2f2ef1e8bf21d08abbf776b7fb6c3600

**Comment by MaxWofford** [comment]
@saescapa Thanks for your contribution!

File: case_studies/san_mateo_high_school/01_first_meet/recap.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- The VGA cable is "strange" with an extra piece of metal which makes it not fit
+ to the adapter.
+  - We ended up transferring the presentation to the teacher's computer and
+  present from there.
+  - The internet had gone down the second I started uploading, we needed to use
+  a usb stick.
+  - Presentation Delayed 1 min.
+- A lot of hackers came in, a total of 38 people came in a room for 30. Some
+people needed to stand in the back.
+- All Leaders became a little nervous but we managed it properly.
```
Comment (jonleung) [inline case_studies/san_mateo_high_school/01_first_meet/recap.md]: @saescapa Curious to hear how did you manage it properly?

File: case_studies/san_mateo_high_school/02_club_fair/recap.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+## Problems we faced
+
+- The windy environment was a problem, as we needed to hold the papers tight.
+
+## Tips
+
+- Having a Big Poster I think influenced a lot of people to come by and check it
+ out.
+- If you do share a table, make sure to keep organized.
+- Make sure to have in your poster the most cool stuff and familiar logos that
```
Comment (jonleung) [inline case_studies/san_mateo_high_school/02_club_fair/recap.md]: Good point!

File: case_studies/san_mateo_high_school/02_club_fair/recap.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+will attract people.
+- Talk to people as they come by, say Hi to everyone, some of them will come and
+ visit.
+- Talk about what you will be doing during the school year briefly, and then
+talk about very exciting project people seemed interested in such as hackathons.
+- Don't expect _anybody_ to know about programming.
+- Shoutouts is a great way to advertise your club, make sure to not get
```
Comment (jonleung) [inline case_studies/san_mateo_high_school/02_club_fair/recap.md]: What is a shoutout?

**Comment by saescapa** [comment]
I added some stuff to clarify on the question you made @jonleung Thank you! Going to PR soon.

**Comment by jonleung** [comment]
:+1:

**Pull request closed without merging**